graphql/permissions_resolver.go

```diff
@@ -79,6 +79,31 @@ func (r *permissionsResolver) ProjectPermissions(ctx context.Context, obj *Permi
 	}, nil
 }
 
+// RepoPermissions is the resolver for the repoPermissions field.
+func (r *permissionsResolver) RepoPermissions(ctx context.Context, obj *Permissions, options RepoPermissionsOptions) (*RepoPermissions, error) {
+	usr, err := user.FindOneById(obj.UserID)
+	if err != nil {
+		return nil, ResourceNotFound.Send(ctx, "user not found")
```

Comment:
This seems like an `InternalServerError` since `FindOneById` would only error if the DB produced an error other than not found.